addons/website_profile/views/website_profile.xml

```diff
@@ -330,24 +330,18 @@
                 </linearGradient>
             </svg>
             <div class="o_pc_overlay d-flex flex-column align-items-center justify-content-center p-4">
-                <t t-set="rank_id" t-value="next_rank_id if next_rank_id else user.rank_id"/>
-                <div t-field="rank_id.image_1920"
+                <div t-field="(next_rank_id or user.rank_id).image_1920"
                     t-attf-style="max-width: {{img_max_width or '50%'}};"
                     t-options="{'widget': 'image', 'preview_image': 'image_128'}"/>
-                <h4 class="text-center" t-field="rank_id.name"/>
-                <small class="w-50 text-center">
-                    <t t-set="karma_value" t-value="next_rank_id.karma_min - user.karma if next_rank_id else user.karma"/>
-                    <t t-if="next_rank_id">
-                        Get
-                        <span class="fw-bold text-primary" t-out="karma_value" t-options="{'widget': 'integer', 'format_decimalized_number': True, 'precision_digits': 2}"/> xp
-                        to level up!
-                    </t>
-                    <t t-else="">
-                        <span class="fw-bold text-primary" t-out="karma_value" t-options="{'widget': 'integer', 'format_decimalized_number': True, 'precision_digits': 2}"/> xp
-                    </t>
-                </small>
             </div>
         </div>
+        <div t-if="next_rank_id" class="text-center">
```

Comment:
Unfortunately, this kind of diff is not allowed as per our stable policy. If we change the arch/DOM, overrides from the community may break. Unless it is necessary to avoid a crash, we usually don't do this in stable.
Would there be another way to achieve a similar result? (Of course we can then merge an IMP/CLN in master).